src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <img
```

Comment:
you should not add empty lines if elements are on the different levels of nesting
```suggestion
      method="post"
    >
      <img
```

Reply:
Thank you for your feedback.
now , it is updated. Please check it !

Reply:
@naziamoff